test/examples/TrafficLight.test.js

```diff
@@ -0,0 +1,24 @@
+var assert = require('assert');
+describe("Traffic Light", function() {
+    var include = require('../helpers/Utils').include;
+    include('Core.js');
+    Core.registerEventPoint('DOM_Init'     , {log: false});
+    include('examples/states/trafficLight/Car.js');
+    include('examples/states/trafficLight/TrafficLight.js');
+
```

Comment:
Why do you check a correct work of cars and traffic lights if this is a unit test for trafficlight?
I think we should consider it functional test. 

The first thing. In this example we have a traffic light and cars. Let's make test cases more human readable. For example:
`cars should move after Traffic Light state becomes green`
`cars should not move after Traffic Light state becomes red`
And name it something like... `trafficlight-cars-general.func.test.js`.

The second thing is that you use setTimeout in your cases. There I'd propose to make your tests cases look like:
`cars should start moving in 3000 after traffic light state becomes red` or something like that. And it can be another functional test. For example `trafficlight-cars-timing.func.test.js`.